lib/internal/fs/utils.js

```diff
@@ -731,11 +747,12 @@ const validatePath = hideStackFrames((path, propName = 'path') => {
   );
 });
 
-const getValidatedPath = hideStackFrames((fileURLOrPath, propName = 'path') => {
-  const path = toPathIfFileURL(fileURLOrPath);
-  validatePath(path, propName);
-  return path;
-});
+const getValidatedPath =
```

Comment:
Can you add a JSDoc and a documentation to this function as well?

Reply:
Fixed.